test/functional/services/common/browser.ts

```diff
@@ -632,8 +633,33 @@ class BrowserService extends FtrService {
     return Boolean(result?.state === 'granted');
   }
 
-  public getClipboardValue(): Promise<string> {
-    return this.driver.executeAsyncScript('navigator.clipboard.readText().then(arguments[0])');
```

Comment:
Unrelated to the main change, but I thought this update doesn't worth a separate PR